### PR TITLE
stimpacks keep you out of crit

### PR DESCRIPTION
**Pull request by ShadowDeath6**
I HAVE ASCENDED, NOT EVEN YOUR PITIFUL LASERS CAN STOP ME NOW

stimpacks will keep you from dropping unconscious from crit, you effectively have 200 health while they're in your system (also you can't be slowed down :^) )

if you go over 200 damage you're fucked though

price has been increased to reflect this, I doubt it's still even as strong as adrenals 

:cl: ShadowDeath6
rscadd: Syndicate scientists have engineered a neural stimulant and added it to their stimpacks, which allows their agents to push their bodies to their absolute limits. Price has been increased to reflect this upgrade.
/:cl:

### CODE REVIEW

**Comment by Super3222** [comment]
I don't like the healing bit. I feel like being able to survive after crit is a buff enough to be happy with.

And 10 tc does not equate to having 200 health, regen factor, stun reduction, and a speed buff.
Even if you have a random chance of dropping dead instantly, it equates to a very shitty traitor item which from my perspective, will only be used to murderbone. But, correct me if I'm wrong.

**Comment by ShadowDeath6** [comment]
it was already only used for combat, this just makes it strong when sec starts using lethals on you. I'll remove the 3x healing and just have it continually heal you rather than having it only heal if you're at a certain health, which is kind of dumb for a traitor medicine injectable

**Comment by EMT321** [comment]
Why is this needed?

**Comment by ShadowDeath6** [comment]
because it's interesting and it was kind of a useless traitor item

**Comment by ShadowDeath6** [comment]
well now that that's all balanced I'm gonna go to sleep

**Comment by EMT321** [comment]
This is a significant buff. No other item to my knowledge allows you to double your health.

**Comment by ghost** [comment]
Honestly, this is interesting...

The inability to enter crit-IE, you'd have about the same effective health as a borg, while being stunnable, means that you can still be beaten when on stimulants. And even when you're tripping balls on stims and you're running around with 200 health, you still retain the risk of dropping into crit instantly once the time passes.

See, you could worry that you could just keep spamming patches, etc. And being able to withstand 4 point blank shots to the chest from a combat shotgun is, well, powerful to say the least. But on the other hand even a weak poison will be enough to crit the guy since they can't antitox themselves without losing their buff.

I'd say make it obvious they're on stims-making them jitter, emit custom emotes, something, so that people know what they're up against.

**Comment by ShadowDeath6** [comment]
I'll have it jitter and emote when they're below the threshold

**Comment by ShadowDeath6** [comment]
credit to super for like all those lines

**Comment by Super3222** [comment]
:ok_hand: I still find it a bit weird that they're still getting a health increase ontop of a speed boost and stun reduction but, I guess it's all or nothing if you're buying this.

**Comment by ShadowDeath6** [comment]
it's 10 tc you butt

**Comment by ParagonPrime** [comment]
> custom emotes

inb4 the janitor is suddenly an expert in the side effects of syndicate stimpacks and screams "TATOR" as soon as he sees a random guy shiver or something

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -809,6 +810,14 @@
 	M.AdjustStunned(-3, 0)
 	M.AdjustWeakened(-3, 0)
 	M.adjustStaminaLoss(-5*REM, 0)
+	if(M.health < 100)//once you start dropping below crit it's obvious you're on stims
```
Comment (X-TheDark) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Pretty sure it's `if(M.health < 0)`, since health is measured from -100 to 100.

Comment (ShadowDeath6) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: oh shit you're right, I knew that, I just forgot

**Comment by ghost** [comment]
@ParagonPrime stims after this will be assumed to be one of the traitor items syndies are known for, alongside eswords, cryptosequencers, c4, etc.

**Comment by ParagonPrime** [comment]
i'm on the fence there, since it seems pretty ridiculous that people won't give a fuck about someone hopped up on meth and ephedrine but as soon as someone has THAT SPECIFIC EMOTE that is tied to THAT SPECIFIC CHEM they're a SYNDIE REEEEEEEE

**Comment by ShadowDeath6** [comment]
you only do the emote if you're crit, so it's safe to assume that they're syndie anyway since they're taking 8 lasers to the face and not dropping

**Comment by ShadowDeath6** [comment]
this is r4m

**Comment by Super3222** [comment]
After -100 usually, someone would be bleeding out / slowly dying while in shock. However, in this case the person is losing their marbles, their brain is jumping all over the place, and they can't keep thoughts from their subconscious inside of their head.

Hence all of that "HEIL SYNDICATE", "Fuck, I knew i should've stuck to cleaning the floor", "Dad? Are you there?" and other weird shit is caught up in there. They're slowly being driven into insanity.

**Comment by ParagonPrime** [comment]
okay i'm totally fine with it if the message only shows once the person is supposed to already be in crit, that's fair enough

a la person with 6 gunshot wounds, bleeding heavily in 10 different places, still stumbling along looking coked up you can probably assume they're on drugs

my issue is only if you can INSTANTLY TELL APART STIMPAKS FROM ""legal"" (aka non-validsalid) drugs with no other indicator

**Comment by ShadowDeath6** [comment]
it has no other indicators when you're not meming below crit

File: code/game/gamemodes/changeling/powers/fakedeath.dm [hunk 2]
```diff
@@ -15,8 +15,7 @@
 	user.update_stat()
 	user.update_canmove()
 	if(user.stat != DEAD)
-		user.emote("deathgasp")
-		user.tod = worldtime2text()
+		user.death()
```
Comment (Cruix) [inline code/game/gamemodes/changeling/powers/fakedeath.dm:18]: This will set your stat var to DEAD when it should be FAKEDEATH

Comment (ShadowDeath6) [reply code/game/gamemodes/changeling/powers/fakedeath.dm:18]: that's... literally the point, they were registering as alive on medhuds and scanners

Comment (Cruix) [reply code/game/gamemodes/changeling/powers/fakedeath.dm:18]: Apparently FAKEDEATH is a status flag now, not a stat.

**Comment by Super3222** [comment]
@Cruix Are we doing a testmerge this time WITHOUT actually merging?

**Comment by Cruix** [comment]
I didn't mark it as merge ready and I forgot after I merged all the other stuff

**Pull request merged**